packages/theme/pages/MyAccount.vue

```diff
@@ -4,15 +4,37 @@
       class="breadcrumbs desktop-only"
       :breadcrumbs="breadcrumbs"
     />
-    <div class="my-account">
+    <SfTabs
+      :openTab="1"
+      tabShowText="show"
+      tabHideText="hide"
+      class="smartphone-only"
+    >
+      <SfTab :title="$t('pages.my_account.content_page_title_my_profile')">
+        <MyProfile />
+      </SfTab>
+      <SfTab :title="$t('pages.my_account.content_page_title_saved_addresses')">
+        <SavedAddressesDetails />
+      </SfTab>
+      <SfTab :title="$t('pages.my_account.content_page_title_order_history')">
+        <OrderHistory />
+      </SfTab>
+      <SfMenuItem
+        :label="$t('pages.my_account.content_page_title_log_out')"
+        class="sf-accordion-item__header"
+        @click="handleLogout"
+      />
+    </SfTabs>
+
+    <div class="my-account desktop-only">
       <div class="my-account__sidebar">
         <h1>{{ $t('pages.my_account.content_page_title_my_account') }}</h1>
         <h2>{{ $t('pages.my_account.content_category_title_personal_details') }}</h2>
-        <NuxtLink to="/my-account">{{ $t('pages.my_account.content_page_title_my_profile') }}</NuxtLink>
+        <NuxtLink to="/my-account/my-profile">{{ $t('pages.my_account.content_page_title_my_profile') }}</NuxtLink>
         <NuxtLink to="/my-account/saved-addresses">{{ $t('pages.my_account.content_page_title_saved_addresses') }}</NuxtLink>
         <h2>{{ $t('pages.my_account.content_category_title_order_details') }}</h2>
         <NuxtLink to="/my-account/order-history">{{ $t('pages.my_account.content_page_title_order_history') }}</NuxtLink>
-        <span @click="handleLogout">{{ $t('pages.my_account.content_page_title_log_out') }}</span>
+        <span class="my-account__logout" @click="handleLogout">{{ $t('pages.my_account.content_page_title_log_out') }}</span>
```

Comment:
Any reason for not converting the \<span> element into \<button>?

Reply:
Correct, this should be a button.